src/engine/DrawManager.java

```diff
@@ -2725,25 +2725,34 @@ public void animateLoading(int x, int y){
 	 * @param separationLineHeight
 	 * @param lives
 	 */
-	public void drawBackground(final Screen screen, int separationLineHeight, int lives){
+	public void drawBackground(final Screen screen, int separationLineHeight, int lives, int level){
+
 		int height = screen.getHeight();
 		int width = screen.getWidth();
 
-		if (bgTimer.checkFinished()){
-			brightness+= lighter;
-			if (brightness >= 70) lighter *= -1;
-			else if (brightness <= 0) lighter *= -1;
-			bgTimer.reset();
+		if (level == 8) {
+			backBufferGraphics.setColor(Color.black);
+			backBufferGraphics.drawRect(0, separationLineHeight, width, height);
 		}
+		else {
+			if (bgTimer.checkFinished()){
```

Comment:
[P3]
I also think using 'else if{}' is a good idea.

Reply:
Actually, the else and if cannot be merged as then it would be different code.